### DISCUSSION TITLE
Help With Connecting to DBs

### DISCUSSION THREAD

**Discussion by lazymman345** [OwlGamingCommunity/V#19] [Support]
Hi Guys,

I've gotten up to the point where I am trying to launch the server but am getting errors thrown when attempting to connect to the MySQL DBs. 

This is my first time working on a project like this so attempting to learn as I go along. 

I've downloaded XAMPP and imported the DBs to myPHPadmin. I've changed the ports in the TODO_GITHUB section to all (4) be 3306, however when attempting to run the server, it seems like it is still trying to connect to port 3307. 

![image](https://user-images.githubusercontent.com/90817333/133722541-da75a0a6-c9c4-429a-b928-5ceaa9529ca4.png)
![image](https://user-images.githubusercontent.com/90817333/133722555-45b56c9d-5a00-4d9b-a86f-e73679a337f6.png)
![image](https://user-images.githubusercontent.com/90817333/133722758-9ec8d4de-c319-4af1-9dba-29ebb68ae0ac.png)

Here myPHPadmin config: 

![image](https://user-images.githubusercontent.com/90817333/133722893-2caf6fda-69da-4746-8376-604a35b9744a.png)

And the lines changed in both the MySQL.cs and ThreadedMySQL.cs files 

![image](https://user-images.githubusercontent.com/90817333/133722969-338f60fd-5e25-4459-9f7b-56aff1708949.png)
![image](https://user-images.githubusercontent.com/90817333/133722979-b2198b6c-73bf-4a43-b2fd-60d28e9ca2fb.png)

**Comment by Bullsito** [comment]
Hi, how do you even manage to get the 'MySQL Initialized'? 

We spent almost a month with my team trying to stablish connection and we only get the SSL error or the credentials error without even get connected.  (I even spam this github, forum and their discord and nobody could give me an answer)

Anyway, did you recompile when you change the ports?


Would you be so kind to share the whole new project (if there is no private work there, of course)? We're struggling to make it functional

Idk if you fix it or maybe unrelevant but:

![image](https://user-images.githubusercontent.com/77119291/133834979-b4ec28f1-9d83-49a0-87bb-41cfefb13882.png)

**Reply by Yannick-DG** [reply]
Would be nice to see an error message too.

**Comment by Yannick-DG** [comment]
> Hi Guys,
> 
> I've gotten up to the point where I am trying to launch the server but am getting errors thrown when attempting to connect to the MySQL DBs.
> 
> This is my first time working on a project like this so attempting to learn as I go along.
> 
> I've downloaded XAMPP and imported the DBs to myPHPadmin. I've changed the ports in the TODO_GITHUB section to all (4) be 3306, however when attempting to run the server, it seems like it is still trying to connect to port 3307.
> 
> ![image](https://user-images.githubusercontent.com/90817333/133722541-da75a0a6-c9c4-429a-b928-5ceaa9529ca4.png) ![image](https://user-images.githubusercontent.com/90817333/133722555-45b56c9d-5a00-4d9b-a86f-e73679a337f6.png) ![image](https://user-images.githubusercontent.com/90817333/133722758-9ec8d4de-c319-4af1-9dba-29ebb68ae0ac.png)
> 
> Here myPHPadmin config:
> 
> ![image](https://user-images.githubusercontent.com/90817333/133722893-2caf6fda-69da-4746-8376-604a35b9744a.png)
> 
> And the lines changed in both the MySQL.cs and ThreadedMySQL.cs files
> 
> ![image](https://user-images.githubusercontent.com/90817333/133722969-338f60fd-5e25-4459-9f7b-56aff1708949.png) ![image](https://user-images.githubusercontent.com/90817333/133722979-b2198b6c-73bf-4a43-b2fd-60d28e9ca2fb.png)

Looks like you did not set an environment variable for your database password.
